concourse/scripts/pipelines-cloudfoundry.sh

```diff
@@ -14,6 +14,7 @@ generate_vars_file() {
    cat <<EOF
 ---
 aws_account: ${AWS_ACCOUNT:-dev}
+auto_deploy: $([ "${ENABLE_AUTO_DEPLOY:-}" ] && echo "true" || echo "false")
```

Comment:
I got an error when trying to deploy the pipeline using either option:
```
➜  paas-cf git:(enable_build_from_master_113265403) ✗ BRANCH=$(git rev-parse --abbrev-ref HEAD) ./concourse/scripts/pipelines-cloudfoundry.sh
Concourse API target dcarley
Deployment dcarley
Pipeline deploy-cloudfoundry
Config file /Users/dcarley/projects/paas/paas-cf/concourse/scripts/../pipelines/deploy-cloudfoundry.yml
failed to parse configuration file: yaml: unmarshal errors:
  line 85: cannot unmarshal !!str `false` into bool
➜  paas-cf git:(enable_build_from_master_113265403) ✗ ENABLE_AUTO_DEPLOY=1 BRANCH=$(git rev-parse --abbrev-ref HEAD) ./concourse/scripts/pipelines-cloudfoundry.sh
Concourse API target dcarley
Deployment dcarley
Pipeline deploy-cloudfoundry
Config file /Users/dcarley/projects/paas/paas-cf/concourse/scripts/../pipelines/deploy-cloudfoundry.yml
failed to parse configuration file: yaml: unmarshal errors:
  line 85: cannot unmarshal !!str `true` into bool
```

My local shell is zsh, if that turns out to matter.